app/assets/stylesheets/components/_search_filter.scss

```diff
@@ -1,19 +1,36 @@
 .search-filter {
-  border-radius: $search-filter-radius;
+  border-top-left-radius: $search-filter-radius;
+  border-top-right-radius: $search-filter-radius;
   border: 1px solid  $search-filter-border-color;
   overflow: hidden;
 }
 
+.svg-icon-tooltip {
+  width: $further-info-icon-width;
```

Comment:
These colour variables should be renamed to something more generic if we're borrowing them here